Scripts/Python/plasma/Plasma.py

```diff
@@ -1270,6 +1289,27 @@ def stop(self):
         """Stops the animation"""
         pass
 
+class ptAsyncTask(Awaitable):
```

Comment:
Only noticed this today, but my first suggestion here was wrong, sorry. You *have* to explicitly re-state the type parameter, otherwise it's assumed to be `Any`. See the [`typing` docs](https://docs.python.org/3/library/typing.html#user-defined-generic-types):

> Using a generic class without specifying type parameters assumes [`Any`](https://docs.python.org/3/library/typing.html#typing.Any) for each position. In the following example, `MyIterable` is not generic but implicitly inherits from `Iterable[Any]`:
> 
> ```python
> from collections.abc import Iterable
> 
> class MyIterable(Iterable): # Same as Iterable[Any]
> ```

So this should be:

```suggestion
class ptAsyncTask(Awaitable[T_co]):
```

By the way, I think you're also missing a definition for `T_co`. (The `typing` module has its own definition of `T_co`, but it's undocumented and the code says that it's "not for export".)